Makefile

```diff
@@ -64,24 +64,20 @@ benchmark-python-local:
 	FEAST_USAGE=False IS_TEST=True FEAST_IS_LOCAL_TEST=True python -m pytest --integration --benchmark  --benchmark-autosave --benchmark-save-data sdk/python/tests
 
 test-python:
-	FEAST_USAGE=False \
-	IS_TEST=True \
-	python -m pytest -n 8 sdk/python/tests \
+	python -m pytest -n 8 --color=yes sdk/python/tests
 
 test-python-integration:
-	FEAST_USAGE=False IS_TEST=True python -m pytest -n 8 --integration sdk/python/tests
+	python -m pytest -n 8 --integration --color=yes --durations=5 --timeout=1200 --timeout_method=thread sdk/python/tests
 
 test-python-integration-local:
 	@(docker info > /dev/null 2>&1 && \
-		FEAST_USAGE=False \
```

Comment:
I believe we should keep `FEAST_USAGE` and `IS_TEST`

Reply:
I found that there's being set in [conftest](https://github.com/feast-dev/feast/blob/master/sdk/python/tests/conftest.py#L27) regardless

Reply:
fixed, thanks